apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -49,68 +47,80 @@ function WpcomNux() {
-					{ __(
-						'In the WordPress editor, each paragraph, image, or video is presented as a distinct “block” of content.'
-					) }
+			<GuidePage className="wpcom-block-editor-nux__page">
+				<h1 className="wpcom-block-editor-nux__heading">{ welcomeHeading }</h1>
+				<p className="wpcom-block-editor-nux__text">
+					{ __( 'Create and edit site pages, and customize the look and feel of each page.' ) }
 				</p>
+				<img
+					alt=""
+					aria-hidden="true"
+					className="wpcom-block-editor-nux__image"
+					src={ editorImage }
+				/>
 			</GuidePage>
 
-			<GuidePage className="edit-post-welcome-guide__page">
-				<h1 className="edit-post-welcome-guide__heading">{ __( 'Make each block your own' ) }</h1>
-				<EditorImage className="edit-post-welcome-guide__image" />
-				<p className="edit-post-welcome-guide__text">
+			<GuidePage className="wpcom-block-editor-nux__page">
+				<h1 className="wpcom-block-editor-nux__heading">
+					{ __( 'Create pages and add your content' ) }
+				</h1>
+				<img
+					alt=""
+					aria-hidden="true"
+					className="wpcom-block-editor-nux__image"
+					src={ blockImage }
+				/>
+				<p className="wpcom-block-editor-nux__text">
 					{ __(
-						'Each block comes with its own set of controls for changing things like color, width, and alignment. These will show and hide automatically when you have a block selected.'
+						'Create and rearrange your site pages. Customize your site navigation menus so your visitors can explore your site.'
 					) }
 				</p>
 			</GuidePage>
 
-			<GuidePage className="edit-post-welcome-guide__page">
-				<h1 className="edit-post-welcome-guide__heading">
-					{ __( 'Get to know the block library' ) }
-				</h1>
-				<BlockLibraryImage className="edit-post-welcome-guide__image" />
-				<p className="edit-post-welcome-guide__text">
-					{ __experimentalCreateInterpolateElement(
-						__(
-							'All of the blocks available to you live in the block library. You’ll find it wherever you see the <InserterIconImage /> icon.'
-						),
-						{
-							InserterIconImage: (
-								<InserterIconImage className="edit-post-welcome-guide__inserter-icon" />
-							),
-						}
+			<GuidePage className="wpcom-block-editor-nux__page">
+				<h1 className="wpcom-block-editor-nux__heading">{ __( 'Add anything you want' ) }</h1>
+				<img
+					alt=""
+					aria-hidden="true"
+					className="wpcom-block-editor-nux__image"
+					src={ blockPickerImage }
+				/>
+				<p className="wpcom-block-editor-nux__text">
+					{ __(
+						'Insert text, photos, forms, Yelp reviews, testimonials, maps, and many more types of blocks. Rearrange blocks on your pages to meet your needs.'
 					) }
 				</p>
 			</GuidePage>
 
-			<GuidePage className="edit-post-welcome-guide__page">
-				<h1 className="edit-post-welcome-guide__heading">
-					{ __( 'Learn how to use the WordPress editor' ) }
+			{ /* @TODO: hide for sites that are already public */ }
```

Comment:
This last item can be initially left out as well, to get other changes in and this separately.